client/src/components/Ticketing/checkout/CheckoutPage.tsx

```diff
@@ -58,8 +58,8 @@ export default function CheckoutPage(): ReactElement {
 
   return (
     <div className='bg-zinc-200 flex flex-col md:flex-col sm:flex-col
-         sm:items-center w-full h-full p-20'>
-      <div className='w-full flex flex-row mb-5'>
+         max-md:items center w-full h-full p-2 md:p-20'>
```

Comment:
Is this supposed to be `items center` and not `items-center`?

Reply:
changed to items-center